src/engraving/rw/read400/tread.cpp

```diff
@@ -3051,6 +3051,10 @@ void TRead::read(MMRest* r, XmlReader& e, ReadContext& ctx)
             NoteDot* dot = Factory::createNoteDot(r);
             TRead::read(dot, e, ctx);
             r->add(dot);
+        } else if (tag == "mmRestNumberPos") {
+            // Old property, deprecated in 4.5
+            r->setNumberOffset(e.readDouble() - ctx.style().styleS(Sid::mmRestNumberPos).val());
+        } else if (TRead::readProperty(r, tag, e, ctx, Pid::MMREST_NUMBER_OFFSET)) {
```

Comment:
I don't think we need this. The property is introduced in 4.5 and shouldn't show up in in pre-4.5 scores

Reply:
Well spotted, thanks, I obvs meant to put this in the 410 reader